alienfile

```diff
@@ -21,11 +21,26 @@ share {
 
   plugin 'Download' => (
     url     => 'https://github.com/westes/flex/releases',
-    version => qr/^flex-([0-9\.]+)\.tar\.gz$/,
+    version => 'flex-2.6.4.tar.gz',
   );
 
   plugin 'Extract' => 'tar.gz';  
-  
+
+  patch [ '%{patch} -p1 < %{.install.patch}/flex_2.6.4-6.2.diff' ];
+
+  before build => sub {
+
+    # this monstrosity is to ensure that the files have the correct
+    # relative timestamps, otherwise the build will try to
+    # regenerate things using autoconf, which is not wanted.
+    require File::Touch;
+    for my $file ( 'configure.ac', 'aclocal.m4', 'Makefile.in', 'configure', 'doc/flex.1' ) {
+	File::Touch::touch($file);
+	sleep(1);
```

Comment:
Can we get away with the sleep being after the loop?

Reply:
It's an honest question, I'm fine with keeping this as-is if it makes it work.  I remember having to do similar workarounds for other aliens.

Reply:
Empirically, without some sort of delay between the touches, they fire off so quickly the timestamps aren't unique and the builds fail.  I originally had no sleep, and, also, no luck.

I'll use Time::HiRes to shorten the sleep period.

Reply:
👍 keep it.